superset-frontend/src/dashboard/components/SliceAdder.jsx

```diff
@@ -253,7 +254,7 @@ class SliceAdder extends React.Component {
             value={this.state.sortBy}
             onChange={this.handleSelect}
             options={Object.entries(KEYS_TO_SORT).map(([key, label]) => ({
-              label: t('Sort by %s', label),
+              label: t('Sort by %s', t(label)),
```

Comment:
This doesn't look right; you can't translate variables like this (the original code looks correct)

Reply:
PR: https://github.com/apache/superset/pull/20080